feature_layers/feature-collection-layer-from-portal/README.md

```diff
@@ -0,0 +1,34 @@
+# Feature Collection Layer From Portal
+
+This sample demonstrates opening a feature collection saved as a portal item
+
+![](FeatureCollectionLayerFromPortal.png)
+
+## Use case
+
+Feature collection layers are often used to hold features with mixed geometry or unstructured data. You can display feature collections stored in a Portal, which are often used for showing content from a CSV or map notes.
+
+## How to use the sample
+
+The feature collection is loaded from the Portal item when the sample starts. Enter feature collection saved as portal item in the text field
+and click on button to see result.
```

Comment:
I don't think we need to manually break the lines here, as there's no official character limit per line.
I think we can also rephrase the instruction a tiny bit, because what we're really entering is the ID of the portal item. How about:
```suggestion
The feature collection is loaded from the Portal item when the sample starts. Enter the ID of a Portal item that holds a feature collection into the text field, and click on the button to see result.
```